resources/js/src/base.js

```diff
@@ -211,6 +211,13 @@ Vue.prototype.$ceres = App;
 
 const store = createStore();
 
+if (window.__INITIAL_STATE__)
```

Comment:
This will never be set if you disable SSR.

Reply:
Ok, I removed it.